w3id/.htaccess

```diff
@@ -0,0 +1,37 @@
+Header set Access-Control-Allow-Origin *
+Options +FollowSymLinks
+RewriteEngine on
+
+# Specific content-type -> Access respective ontology
+
+# HTML:
+# RewriteCond %{HTTP_ACCEPT} text/html
+# RewriteRule ^$ https://nfdi4ing.pages.rwth-aachen.de/metadata4ing/metadata4ing/index.html [R=303,L,NE]
+
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^(2020)$ https://terminology.tib.eu/ts/ontologies/dfgfo [R=303,L,NE]
+
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^([2-9]\d{3})$ https://terminology.tib.eu/ts/ontologies/dfgfo$1 [R=303,L,NE]
```

Comment:
As long as we don't have versions of an ontology stored in the TS we should resolve to the ontology PURL here as well.

Reply:
thanks @StroemPhi